app/models/manageiq/providers/ibm_terraform/configuration_manager.rb

```diff
@@ -48,6 +51,7 @@ def self.create_from_params(params, endpoints, authentications)
       endpoints.each { |endpoint| ems.assign_nested_endpoint(endpoint) }
       authentications.each { |authentication| ems.assign_nested_authentication(authentication) }
 
+      ems.ensure_provider_zone_id
       ems.provider.save!
       ems.save!
     end
```

Comment:
Thanks for reminding me, we can actually delete these methods now.  In core we are saving the provider which is the only reason why they had to be overridden

Reply:
https://github.com/ManageIQ/manageiq/pull/20535